scss/modules/_filters.scss

```diff
@@ -20,25 +20,34 @@
 //
 // Styleguide modules.filters
 
+// by default show filters offscreen
 .filters {
   @include clearfix();
   @include u-bg--neutral();
+  @include transition(left 1s ease-in-out);
```

Comment:
Let's speed up the transition and go with ease-out to match the transition of the mobile menu:
`.2s ease-out`